### PR TITLE
Fix lint errors

### LINKED ISSUE #14711: `jake lint` isn't linting everything
`jake lint` and `gulp lint` don't seem to be catching all (any?) lint errors.
The following command *will* work:

```
node_modules/tslint/bin/tslint 'src/**/*.ts' --exclude 'src/lib/*.generated.d.ts' --exclude 'src/harness/unittests/services/formatting/**/*.ts' --exclude 'src/harness/unittests/services/formatting/*.ts' --format stylish
```

Why don't we just use that? Tslint is fast enough now that we don't really need a parallelized linter.

### PR DESCRIPTION
**Pull request by andy-ms**
Fixing some new lint errors I found.
Stopping these errors from coming back is #14711.
**Update**: Now fixes #14711 too.

### CODE REVIEW

File: src/lib/es5.d.ts [hunk 1]
```diff
@@ -2,6 +2,8 @@
 /// ECMAScript APIs
 /////////////////////////////
 
+// tslint:disable no-var-keyword
```
Comment (mhegazy) [inline src/lib/es5.d.ts]: not sure i agree that we should ship our library with these comments. i would rather we exclude them from the lint.

Comment (mhegazy) [inline src/lib/es5.d.ts]: Otherwise 👍

Comment (andy-ms) [inline src/lib/es5.d.ts]: Can we change those to `const`?

Comment (andy-ms) [inline src/lib/es5.d.ts]: Removed this file from linting for now.

**Pull request merged**